src/Definition/Repository/Cache/Compiler/AttributesCompiler.php

```diff
@@ -91,7 +91,8 @@ private function compileNativeAttribute(ReflectionAttribute $reflectionAttribute
 
         /** @infection-ignore-all */
         if (count($arguments) > 0) {
-            $arguments = var_export($arguments, true);
+            $arguments = serialize($arguments);
+            $arguments = 'unserialize(' . var_export($arguments, true) . ')';
```

Comment:
`serialize` returns a string, no need to `var_export` it. 🙂 

```suggestion
            $arguments = "unserialize('" . serialize($arguments) . "')";
```